charts/xui-webapp/values.preview.template.yaml

```diff
@@ -22,6 +22,7 @@ nodejs:
      FEATURE_WORKALLOCATION_ENABLED: true
      SERVICES_LOCATION_REF_API_URL: http://rd-location-ref-api-aat.service.core-compute-aat.internal
      FEATURE_ACCESS_MANAGEMENT_ENABLED: true
+     NODE_OPTIONS: --max-old-space-size=8192
```

Comment:
So this will increase the memory on each node to 8GB, does the infra support this?